pkg/agent/controller/networkpolicy/priority.go

```diff
@@ -96,24 +73,60 @@ type priorityAssigner struct {
 	ofPriorityMap map[uint16]types.Priority
 	// sortedPriorities maintains a list of sorted Priorities currently registered in the table.
 	sortedPriorities types.ByPriority
-	// initialOFPriorityFunc determines the initial OpenFlow priority to be checked for input Priorities.
-	initialOFPriorityFunc InitialOFPriorityGetter
-	// isSingleTier keeps track of if the priorityAssigner is responsible for handling more than one Tier in
-	// the OVS table that it manages.
-	isSingleTier bool
+	// isBaselineTier keeps track of if the priorityAssigner is responsible for handling the baseline Tier
+	// table (which is shared with K8s NetworkPolicy default tables) or the Antrea Policy tables.
+	isBaselineTier bool
+	// policyBottomPriority keeps track of the lowest ofPriority allowed for flow creation in the table it manages.
+	policyBottomPriority uint16
+	// policyTopPriority keeps track of the highest ofPriority allowed for flow creation in the table it manages.
+	policyTopPriority uint16
 }
 
-func newPriorityAssigner(initialOFPriorityFunc InitialOFPriorityGetter, isSingleTier bool) *priorityAssigner {
+func newPriorityAssigner(isBaselineTier bool) *priorityAssigner {
```

Comment:
have you determined that it was no longer necessary to have the notion of a configurable `InitialOFPriorityGetter`?

Reply:
Yes. Based on recent changes to the file and the new OVS pipeline, I don't feel there's a need for that struct since the PriorityGetter function is straightforward enough.